ml-agents/mlagents/utils/load_from_hf.py

```diff
@@ -0,0 +1,36 @@
+import os
+import logging
+import argparse
+from huggingface_hub import HfApi, snapshot_download
+
+def load_from_hf(repo_id: str, 
+                 local_dir: str):
+    """
+    Download a model from Hugging Face Hub.
+    :param repo_id: id of the model repository from the Hugging Face Hub
+    :param local_dir: local destination of the repository
+    """
+    _, repo_name = repo_id.split('/')
+
+    local_dir = os.path.join(local_dir, repo_name)
+
+    repo_path = snapshot_download(repo_id = repo_id,
+                                  local_dir= local_dir,
+                                  local_dir_use_symlinks="auto")
+
+    logging.info(f"The repository {repo_id} has been cloned to {local_dir}")
```

Comment:
```suggestion
    logging.info(f"The repository {repo_id} has been downloaded to {local_dir}")
```

I would rather say "downloaded" than "cloned" to avoid confusion with a real `git clone`.